core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -258,8 +258,15 @@ object SparkEnv extends Logging {
       if (rpcEnv.isInstanceOf[AkkaRpcEnv]) {
         rpcEnv.asInstanceOf[AkkaRpcEnv].actorSystem
       } else {
+        val actorSystemPort = if (port == 0) 0 else rpcEnv.address.port + 1
         // Create a ActorSystem for legacy codes
-        AkkaUtils.createActorSystem(actorSystemName, hostname, port, conf, securityManager)._1
+        AkkaUtils.createActorSystem(
```

Comment:
Digressing a bit... but if we don't have already, we should file a bug to deprecate and remove `StreamingContext.actorStream` in the 2.0 time frame.

If could be moved to the akka rpc module, for example, if people really need it.